exercises/G-conditionals-2/exercise-2.js

```diff
@@ -7,7 +7,13 @@
 
 */
 
-function studentPassed(grade) {}
+function studentPassed(grade) {
+  if (grade < 50) {
+    return "failed";
+  } else if (grade >= 50) {
```

Comment:
Again, you don't need the check `grade >= 50)` - just else will also work.

Reply:
Thanks for the feedback on the exercises Karen, I have made the changes as mentioned.